index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+
+  <section id="header">
+    <nav class="navbar">
+      <div>
+        <img src="image/My_Logo.png" alt="My Logo">
+      </div>
+      <div class="d-navbar">
+        <a class="a-navbar" href="#Portfolio">Portfolio</a>
+        <a class="a-navbar" href="#About">About</a>
+        <a class="a-navbar" href="#Contact">Contact</a>    
+      </div>
+    </nav>
+  </section>
+  
+  <section id="headline">
+        <h1 class="welcome">I am Paka <br> Glad to see you!</h1>
```

Comment:
Awesome job using a  `h1` for our main heading :partying_face:  , It really is appropriate. You are a Html Ninja :+1: :+1: . 
Did you know we can only use this h1 once and for our most important heading. You did a good job here.